src/pages/index.js

```diff
@@ -6,6 +6,111 @@ import Layout from '../components/layout/layout'
 import Modal from '../components/elements/modal'
 import SignupModal from '../components/elements/signup-modal'
 import { localize, WithIntl } from '../components/localization'
+import Carousel from '../components/elements/carousel'
+import styled from 'styled-components'
+import { Header } from '../components/elements/typography.js'
+import CarouselBackground from 'images/svg/abstract.svg'
+import ChecklistLogo from 'images/svg/checklist.svg'
+
+const HomeCarouselWrapper = styled.div`
+    background-color: var(--color-grey-1);
+    display: flex;
+`
+const HomeCarouselContainer = styled.div`
+    width: 90%;
+    margin-left: 20.8rem;
+    margin-right: 0;
+    display: flex;
+    max-width: 123.2rem;
+    padding: 5.8rem 0 5.4rem 0;
+    padding-top: 5.8rem;
+    padding-bottom: 5.4rem;
+    justify-content: space-between;
+    align-items: flex-start;
+
+    .context-container {
+        margin-top: 3.8rem;
+        color: var(--color-black-2);
+        width: 32.8rem;
+        margin-right: 10.7rem;
+    }
+
+    button {
+        width: 13.5rem;
+        margin-top: 4rem;
+    }
+`
+const Slide1 = styled.section`
+    div {
+        display: inline-flex;
+        margin-top: 4rem;
+        width: 100%;
+        position: relative;
+        animation-name: test;
+        animation-duration: 0.3s;
+        animation-timing-function: linear;
+
+        p {
+            margin-left: 1.6rem;
+            height: 30px;
+            font-size: 20px;
+            color: var(--color-black-2);
+            line-height: 1.5;
+        }
+    }
+    @keyframes test {
+        0% {
+            margin-left: 12.6rem;
+            opacity: 0;
+        }
+        70% {
+            opacity: 0.2;
+        }
+        100% {
+            margin-left: 0;
+            opacity: 1;
+        }
+    }
+    div:first-child {
+        margin-top: 0;
+    }
+`
+const Carousel1 = () => (
+    <Slide1>
+        <div>
+            <ChecklistLogo />
+            <p>Patented, licensed, and regulated</p>
```

Comment:
missing translation + to all the other text also

Reply:
done